BigQuery/src/BigQueryClient.php

```diff
@@ -856,6 +856,29 @@ public function geography($value)
         return new Geography($value);
     }
 
+    /**
+     * Create a BigQuery Json object.
+     *
+     * Json represents a value with a data type of
+     * [JSON](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#json_type)
+     *
+     * Example:
+     * ```
+     * use Google\Cloud\BigQuery\BigQueryClient;
+     *
+     * $bigQuery = new BigQueryClient();
+     * $json = $bigQuery->json('{"key":"value"}');
+     * ```
+     *
+     * @param string|\JsonSerializable|int|null $value The JSON string value.
```

Comment:
Should this be an array instead of int?

Reply:
int can be send actually, array is also required.

Reply:
If it goes so long, then do you think `mixed` makes more sense here?

Reply:
Otherwise, it may look like this (and I may've missed some types here):
```suggestion
     * @param string|\JsonSerializable|int|array|bool|null $value The JSON string value.
```

Reply:
hmm.. I liked the `mixed`. We will go with that

Reply:
Closing this discussion as we are moving to support only the string format